libs/vm_modules/include/vm_modules/math/math.hpp

```diff
@@ -40,6 +40,8 @@ static void CreateAbs(fetch::vm::Module &module)
   module.CreateFreeFunction<int32_t>("Abs", &Abs<int32_t>);
   module.CreateFreeFunction<float_t>("Abs", &Abs<float_t>);
   module.CreateFreeFunction<double_t>("Abs", &Abs<double_t>);
+  module.CreateFreeFunction<fixed_point::fp32_t>("Abs", &Abs<fixed_point::fp32_t>);
+  module.CreateFreeFunction<fixed_point::fp64_t>("Abs", &Abs<fixed_point::fp64_t>);
```

Comment:
we should unify our modules_style. "Abs" here but "sqrt" and "pow" elsewhere.

Reply:
agreed, I would go for lower-case everywhere, ie "abs".